src/App/Homepage.js

```diff
@@ -68,13 +70,28 @@ class Homepage extends Component {
     this.resultListItem.scrollToElement(index);
   }
 
+  handleCategorySelection = (categories) => {
+        console.log(categories);
+
+        if(categories.length === 0 ) {
+
+            this.setState({orgs: this.originalOrgs})
+
+        } else {
+            let filter_criteria_list = find_in_object(this.originalOrgs, {categoryautosortscript: categories});
```

Comment:
Is `categoryautosortscript` a key that we are in control of ? Changing it to `categoryAutoSortScript` will make it easier to read and keep it consistent 👍